include/utl/parser/csv.h

```diff
@@ -53,6 +53,17 @@ inline void parse_column(cstr& s, T& arg) {
                                             adjust_for_quote + adjust_for_cr));
 }
 
+inline void unescape_quoted_string(std::string& arg) {
+  auto found_at = 0;
+  while ((found_at = arg.find('"', found_at)) != -1) {
```

Comment:
```suggestion
  while ((found_at = arg.find('"', found_at)) != std::string::npos) {
```

Not sure if it's defined to be `-1`.